app.rb

```diff
@@ -0,0 +1,93 @@
+require './module/rental'
+require './module/book'
+require './person'
+require './module/student'
+require './module/classroom'
+require './module/teacher'
+
+class App
+  attr_reader :people, :books, :rentals
+
+  def initialize()
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: #{book.title},  Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      case person
+      when Student
+        puts "[Student] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      when Teacher
+        puts "[Teacher] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
```

Comment:
- [ ] There is a problem with the application when it comes to listing the people option (2) as you can see on the console 👇🏻 the name is displayed at the place of the age and the age is displayed where the Name age is. Please this 👍🏻

![image](https://user-images.githubusercontent.com/79058364/219855035-f4cb5727-73fe-499b-a3c3-82c6f7a175eb.png)

Reply:
This fishy stuff:laughing: Lol! Now corrected and fine